worlds/openrct2/Options.py

```diff
@@ -0,0 +1,912 @@
+    default = 0
+
+class ShopMinimumNausea(Range):
+    """If the shop determines you need a ride with a minimum nausea, this value will be the lowest it can ask for.
+    If this value is higher than the maximum, the generator will assume it is a mistake and set it to 0.
+    """
+    display_name = "Minimum Shop Nausea Requirement"
+    range_start = 0
+    range_end = 4
+    default = 0
+
+class ShopMinimumLength(Range):
+    """If the shop determines you need a ride with a minimum length, this value in meters will be the lowest it 
+    can ask for. If this value is higher than the maximum, the generator will assume it is a mistake and set it to 0.
+    """
+    display_name = "Minimum Shop Length Requirement"
+    range_start = 0
+    range_end = 2500
+    default = 610
+
+class ShopMinimumTotalCustomers(Range):
+    """If the shop determines you need a ride with a minimum total number of customers, this value will be the 
+    lowest it can ask for. If this value is higher than the maximum, the generator will assume it is a mistake and 
+    set it to 0.
+    """
+    display_name = "Minimum Shop Customers Requirement"
+    range_start = 0
+    range_end = 2000
+    default = 0
+
+class ShopMaximumExcitement(Range):
+    """If the shop determines you need a ride with a minimum excitement, this value will be the highest it can ask for.
+    """
+    display_name = "Maximum Shop Excitement Requirement"
+    range_start = 0
+    range_end = 10
+    default = 5
+
+class ShopMaximumIntensity(Range):
+    """If the shop determines you need a ride with a minimum intensity, this value will be the highest it can ask for.
+    """
+    display_name = "Maximum Shop Intensity Requirement"
+    range_start = 0
+    range_end = 7
+    default = 5
+
+class ShopMaximumNausea(Range):
+    """If the shop determines you need a ride with a minimum nausea, this value will be the highest it can ask for.
+    """
+    display_name = "Maximum Shop Nausea Requirement"
+    range_start = 0
+    range_end = 4
+    default = 4
+
+class ShopMaximumLength(Range):
+    """If the shop determines you need a ride with a minimum length, this value will be the highest it can ask for.
+    """
+    display_name = "Maximum Shop Length Requirement"
+    range_start = 0
+    range_end = 2500
+    default = 1250
+
+class ShopMaximumTotalCustomers(Range):
+    """If the shop determines you need a ride with a minimum total number of customers, this value will be the 
+    highest it can ask for.
+    """
+    display_name = "Maximum Shop Total Customers Requirement"
+    range_start = 0
+    range_end = 2000
+    default = 800
+
+class BalanceGuestCounts(OpenRCT2OnToggle):
+    """Attempts to balance the minimum guest requirements to the ride they're attached to. Low throughput rides
+    like Spiral Slides will tend towards the minimum, while high throughput rides like roller coasters will 
+    tend towards the maximum.
+    """
+    display_name = "Randomize Park Values"
```

Comment:
```suggestion
    display_name = "Balance Guest Counts"
```

Forgot to change the display name. :)